data/persons.txt

```diff
@@ -1850,3 +1849,149 @@ ship "Modified Ladybug"
+			"What is this for?"
+			"This should be a private member."
+			"Comments?"
+			"Please remove unnecessary space."
+
+			"Looks good to me."
+			"Is this the best name?"
+			"The standard copyright header was changed recently."
+
+			"Please use the template."
+			"The decision to implement fancy quote is one of the worst in computer history."
+			"	This needs another indent."
+#		EjoThims
+	phrase
+		word
+			"If you want your pull request to not be merged, "
+			"If you prefer your pull request never be merged, "
+			"If you prefer your pull request sits idle, "
+			"If you want your pull request to be closed, "
+		word
+			"make sure you keep adding content to it"
+			"always change unrelated things"
+			"do not ever test it and don't add integration tests. Whatever those even are, they're apparently important"
+			"try your best to avoid respecting existing syntax and conventions"
+			"just forget about it for a while"
+			"use the forbidden word: 'realism'"
+			"bury the new feature in semi-relevant data changes"
+			"do not, under any circumstances, ever be willing to compromise"
+			"make sure to completely ignore what the reviewers tell you to do"
+			"skip over writing an issue about it first"
+		word
+			"!"
+			"."
+			". Trust me! It works."
+			". You shouldn't do those things, though! I'm just kidding."
+
+	ship "Waverider" "Pain of the Proofreader"
+		"never disabled"
+	ship "Waverider" "Regret of the Art Critics"
+		"never disabled"
+	ship "Waverider" "Curse of the Code Herders"
+		"never disabled"
+	ship "Waverider" "Tears of the Choreographers"
+		"never disabled"
+	ship "Waverider" "Blood of the Adjuncators"
+		"never disabled"
+	ship "Waverider" "Madness of the Balancers"
+		"never disabled"
+
+ship "Waverider"
+	sprite "ship/waverider"
+	thumbnail "thumbnail/waverider"
+	attributes
+		category "Light Warship"
+		"cost" 3,000,000
+		"shields" 10000
+		"hull" 7000
+		"required crew" 1
+		"bunks" 1
+		"mass" 150
+		"drag" 1.5
+		"heat dissipation" 1
+		"fuel capacity" 1000
+		"cargo space" 10000
+		"outfit space" 1000
+		"weapon capacity" 1000
+		"engine capacity" 1000
+		weapon
+			"blast radius" 340
+			"shield damage" 3400
+			"hull damage" 1700
+			"hit force" 5000
+	outfits
+		"Particle Cannon" 4
+		"S-970 Regenerator"
+		"Dwarf Core"
+		"LP072a Battery Pack"
+		"Liquid Nitrogen Cooler"
+		"Catalytic Ramscoop"
+		"Tactical Scanner"
+		"Cargo Scanner"
+		"Outfit Scanner"
+		"Asteroid Scanner"
+
+		"A120 Atomic Thruster"
+		"A125 Atomic Steering"
+		"A125 Atomic Steering"
+		"Scram Drive"
+
+		"Emotional Baggage"
+	engine 0 56
+	engine -23 64
+	engine 23 64
+	gun -22.5 -56.5 "Particle Cannon"
+	gun 22.5 -56.5 "Particle Cannon"
+	gun -31.5 -7 "Particle Cannon"
+	gun 31.5 -7 "Particle Cannon"
+	description `The Waverider was a ship that was first requested and never used for five years, seven months and twenty-seven days.`
+	description `	Its disappointment was so immense, it willed itself into being.`
+
```

Comment:
```suggestion

	ship "Waverider" "WaveBuger"
		add attributes
			"required crew" 2

```
here's what I think would be cool for the code one - could also go with 1 extra required crew if you think this isnt subtle enough

Reply:
Or, have two bunks, but required crew of 3, therefore, having it constantly disabled on a 33% chance. :p

Reply:
what 2 bunks? I'm the only official code reviewer, and thats the joke xD
also 33% is too low

Reply:
_I thought we had two code reviewers this whole time. :p_

Reply:
officialy I'm the only one